crates/languages/src/typescript.rs

```diff
@@ -167,6 +167,7 @@ impl LspAdapter for TypeScriptLspAdapter {
     ) -> Result<Option<serde_json::Value>> {
         Ok(Some(json!({
             "provideFormatter": true,
+            "hostInfo": "zed",
```

Comment:
We should include the version number here as well, based on the docs for `hostInfo`.

Reply:
I guess it's optional — I believe both VSCode and IntelliJ do not set a version. 

I can add it if the version number is easily available in this context.